stateproof/worker_test.go

```diff
@@ -577,18 +577,15 @@ func TestSignerDoesntDeleteKeysWhenDBDoesntStoreSigs(t *testing.T) {
 	require.Zero(t, len(s.deletedStateProofKeys))
 }
 
-// TestSigBroacastTwoPerSig checks if each signature is broadcasted twice per period
-// It generates numAddresses and prepares a database with the account/signatures.
-// Then, calls broadcastSigs with round numbers spanning periods and
-// makes sure each account has 2 sigs sent per period if originated locally, and 1 sig
-// if received from another relay.
-func TestSigBroacastTwoPerSig(t *testing.T) {
-	partitiontest.PartitionTest(t)
+// getSignaturesInDatabase sets up the db with signatures
+// makeFromThisNode -1 no, +1 yes, 0 mix
```

Comment:
Not important and not necessary. This is internal test function, and used only here.

Reply:
I tend to agree with Or here. 
Tests should be readable as well, and it would be more readable to see
this line
`getSignaturesInDatabase(t, 10, NonLocalSignaturesOnly)`
than this line
`getSignaturesInDatabase(t, 10, -1)`